test/interface/ode_strip_test.jl

```diff
@@ -15,3 +15,10 @@ sol = solve(prob, Rosenbrock23())
 
```

Comment:
also test an implicit method, `TRBDF2`. Also test that a lazy method does the right thing.